### DISCUSSION TITLE
Bun v1.0.8

### DISCUSSION THREAD

**Discussion by Jarred-Sumner** [oven-sh/bun#6864] [Announcements]
To install Bun v1.0.8:
```bash
curl https://bun.sh/install | bash
# or you can use npm
# npm install -g bun
```

To upgrade to Bun v1.0.8:
```bash
bun upgrade
```

**[Read Bun v1.0.8's release notes on Bun's blog](https://bun.sh/blog/bun-v1.0.8)**

## What's Changed
* docs: back the test/README.md file by @Pedromdsn in https://github.com/oven-sh/bun/pull/6626
* fix `bun link` in workspace package by @dylan-conway in https://github.com/oven-sh/bun/pull/6631
* fix(node:buffer): fix `Buffer.write` stuck by @Hanaasagi in https://github.com/oven-sh/bun/pull/6651
* Backport uWS & usockets changes by @Jarred-Sumner in https://github.com/oven-sh/bun/pull/6649
* docs: fix `react.md` typos by @yamcodes in https://github.com/oven-sh/bun/pull/6656
* Fix failing bun add test by @liz3 in https://github.com/oven-sh/bun/pull/6676
* fixed minor typo workspaces.md by @nxzq in https://github.com/oven-sh/bun/pull/6673
* fix(worker_threads): off by one on threadId  by @jerome-benoit in https://github.com/oven-sh/bun/pull/6671
* Add Drizzle guide by @colinhacks in https://github.com/oven-sh/bun/pull/6698
* fix(runtime): remove env variable limit by @paperdave in https://github.com/oven-sh/bun/pull/6697
* Fix `setTimeout(() => {}, 0)` and align `setImmediate` behavior with Node.js by @Jarred-Sumner in https://github.com/oven-sh/bun/pull/6674
* Implement `mock.module` in bun:test by @Jarred-Sumner in https://github.com/oven-sh/bun/pull/6685
* fix drizzle guide by @jasperkelder in https://github.com/oven-sh/bun/pull/6703
* fix cli create from local template by @pierre-cm in https://github.com/oven-sh/bun/pull/6670
* docs: fixed typo in guides, install and runtime folder by @thapasusheel in https://github.com/oven-sh/bun/pull/6718
* Add test for custom thenables in AsyncLocalStorage and fix flaky test and make expect(fn).toThrow drain unhandled promise rejections by @Jarred-Sumner in https://github.com/oven-sh/bun/pull/6701
* fix(requestIP): fix type cast from binary to text format for IPv4. by @Hanaasagi in https://github.com/oven-sh/bun/pull/6725
* fix: emit close on stdin stream end by @liz3 in https://github.com/oven-sh/bun/pull/6720
* Fix running `Bun.spawn` on Vercel and GCP by @Jarred-Sumner in https://github.com/oven-sh/bun/pull/6724
* feat: Windows + CMake Build System by @Jarred-Sumner in https://github.com/oven-sh/bun/pull/4410
* docs: fix `setup.sh` path in `contributing.md` by @RaisinTen in https://github.com/oven-sh/bun/pull/6745
* add cache and main by @rohanmayya in https://github.com/oven-sh/bun/pull/6679
* dx: Fix setup script and contributing docs by @paperdave in https://github.com/oven-sh/bun/pull/6752
* fix: remove extra parenthesis by @o-az in https://github.com/oven-sh/bun/pull/6775
* Fixed broken link in deflatesync section by @perpetualsquid in https://github.com/oven-sh/bun/pull/6774
* docs: spelling of necessarily by @Smoothieewastaken in https://github.com/oven-sh/bun/pull/6772
* docs(contributing) add ruby dependency by @gamedevsam in https://github.com/oven-sh/bun/pull/6788
* Update contributing.md by @kingofdreams777 in https://github.com/oven-sh/bun/pull/6787
* fix(node:buffer): fix `Buffer.readUintBE` by @Hanaasagi in https://github.com/oven-sh/bun/pull/6770
* [docs] fix ws/pubsub; closed ws don't publish to topic by @RohitKaushal7 in https://github.com/oven-sh/bun/pull/6739
* feat(console): add printer for mapiterator by @liz3 in https://github.com/oven-sh/bun/pull/6778
* More resilient test for fetch body memory leak by @Jarred-Sumner in https://github.com/oven-sh/bun/pull/6794
* fix: Macro segmentation faults by @I-A-S in https://github.com/oven-sh/bun/pull/6756
* fix(fetch) by @cirospaciari in https://github.com/oven-sh/bun/pull/6672
* Fix minify-whitespace `case undefined` by @krk in https://github.com/oven-sh/bun/pull/6808
* peer dependency and semver prerelease bug fixes by @dylan-conway in https://github.com/oven-sh/bun/pull/6814
* Fix 6281 by @krk in https://github.com/oven-sh/bun/pull/6809
* Fix memory leak in `require` by @Jarred-Sumner in https://github.com/oven-sh/bun/pull/6790
* fix: provide nodejs compat for dns methods wrapped with util.promisify by @antongolub in https://github.com/oven-sh/bun/pull/6748
* workflows(bun-mac-x64.yml): Fix typo "bun-obj-darwin-x6" by @Marukome0743 in https://github.com/oven-sh/bun/pull/6819
* chore: build system improvements by @paperdave in https://github.com/oven-sh/bun/pull/6811
* fix(build): add option and auto detect for arch linux by @liz3 in https://github.com/oven-sh/bun/pull/6835
* fix #6831 by @dylan-conway in https://github.com/oven-sh/bun/pull/6859

## New Contributors
* @Pedromdsn made their first contribution in https://github.com/oven-sh/bun/pull/6626
* @yamcodes made their first contribution in https://github.com/oven-sh/bun/pull/6656
* @nxzq made their first contribution in https://github.com/oven-sh/bun/pull/6673
* @jasperkelder made their first contribution in https://github.com/oven-sh/bun/pull/6703
* @thapasusheel made their first contribution in https://github.com/oven-sh/bun/pull/6718
* @RaisinTen made their first contribution in https://github.com/oven-sh/bun/pull/6745
* @rohanmayya made their first contribution in https://github.com/oven-sh/bun/pull/6679
* @perpetualsquid made their first contribution in https://github.com/oven-sh/bun/pull/6774
* @Smoothieewastaken made their first contribution in https://github.com/oven-sh/bun/pull/6772
* @gamedevsam made their first contribution in https://github.com/oven-sh/bun/pull/6788
* @kingofdreams777 made their first contribution in https://github.com/oven-sh/bun/pull/6787
* @RohitKaushal7 made their first contribution in https://github.com/oven-sh/bun/pull/6739
* @I-A-S made their first contribution in https://github.com/oven-sh/bun/pull/6756
* @krk made their first contribution in https://github.com/oven-sh/bun/pull/6808
* @antongolub made their first contribution in https://github.com/oven-sh/bun/pull/6748
* @Marukome0743 made their first contribution in https://github.com/oven-sh/bun/pull/6819

**Full Changelog**: https://github.com/oven-sh/bun/compare/bun-v1.0.7...bun-v1.0.8

<hr /><em>This discussion was created from the release <a href='https://github.com/oven-sh/bun/releases/tag/bun-v1.0.8'>Bun v1.0.8</a>.</em>

**Comment by siguici** [comment]
I just tried to migrate to the latest version but I encountered an error so I tried to run the installation script but I got this error:

bash: line 159: 70208 Illegal instruction (core dumped) IS_BUN_AUTO_UPDATE=true $exe completions &> /dev/null
Run 'bun --help' to get started

**Reply by L3P3** [reply]
Same.

**Reply by Ozair0** [reply]
What about V1.0.9? have you tested that one?

**Reply by L3P3** [reply]
I just ran upgrade, its on 1.0.10 now and works again, thank you. :smile:
(Disclaimer: But meanwhile I have also upgraded from i3-3110M to i7-3940XM)
